### ISSUE TITLE
Can't open VSCode stable anymore

### ISSUE THREAD

**Issue by Maijin** [microsoft/vscode#95008]
- VSCode Version: stable (It is however working on insider)
- OS Version: Mac OS 10.15.4 (19E287)

Steps to Reproduce:

1. Open the VSCode binary or execute `code`

Does this issue occur when all extensions are disabled?: Yes

Here is what I have when running code --verbose
```
code --verbose --disable-extensions
[main 2020-04-11T16:55:04.788Z] Error: connect EACCES /Users/maijin/Library/Application Support/Code/1.44.0-main.sock
    at PipeConnectWrap.afterConnect [as oncomplete] (net.js:1056:14)
[main 2020-04-11T16:55:04.788Z] Lifecycle#kill()
```

**Comment by Maijin** [comment]
If anyone encounter this issue - it's linked to one of the file in  `/System/Volumes/Data/Users/XXX/Library/Application Support/Code` - I rm -Rf it and now it works again. I have a local backup in case there is a specific file you'd want to look at to see @deepak1556 to investigate the bug.

**Comment by deepak1556** [comment]
Thanks for the info! Its most likely the file was locked by a zombie instance of VS Code given the `EACCES ` error. Next time when this happens do a `lsof | grep <vscode-version>-main.sock` and see which process is using it.

There is no actionable item for now, but will keep the bug open for updates.

**Comment by Maijin** [comment]
Thanks yeah the sock were indeed busy which is weird. 

But there is still something to improve here I think. Before posting here I've rebooted my computer without success, check the verbose and nothing was really giving much explanation.

A message/alert to the user when opening VS code and --verbose with what's going on and how to resolve this without nuking the file would be really useful!

**Comment by deepak1556** [comment]
This file will soon be invalid, we are changing the use case to resolve another issue. Although I agree with the idea, its not ideal for the short term. Thanks for the feedback!

**Comment by grantg182** [comment]
Just ran in to the exact same problem. Had a look at the `sock` file, which did not appear when running a grep for it. Instead, I navigated into `/Users/me/Library/Application Support/Code` and found that two files, one of which chucked out the EACCESS error, had changed ownerships to root:staff on my OSX machine. Probably because the last time I ran VSC, I ran it as root to be able to edit some root files. Changed ownership back to `me:staff` and all good, VSC fired straight back up.
